src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -50,14 +51,19 @@ const InputPassword: UISchemaElementComponent<UISchemaElementComponentWithValida
   const value = useValue(uischema);
   const { loading } = useWidgetContext();
   const onChangeHandler = useOnChange(uischema);
-  const label = getTranslation(uischema.translations!, 'label');
   const { translations = [], focus, required } = uischema;
   const {
     attributes,
     inputMeta: { name },
   } = uischema.options;
+  const label = getTranslation(translations, 'label');
+  const hint = getTranslation(translations, 'hint');
+  const explain = getTranslation(translations, 'bottomExplain');
   const focusRef = useAutoFocus<HTMLInputElement>(focus);
   const hasErrors = typeof errors !== 'undefined';
+  const hintId = hint && `${name}-hint`;
+  const explainId = explain && `${name}-explain`;
+  const updatedDescribedByIds = buildInputDescribedByValue(describedByIds, hintId, explainId);
```

Comment:
I'd inline the `buildInputDescribedByValue` logic. It doesn't contain any special logic that needs modularization or testing AFAICT... or rename it to something more generic if it's useful to have that filter + join operation for this kind of string building. Seems like something that would be covered by `emotion` which we bring in anyway?

Reply:
Based on the feedback [here](https://okta.slack.com/archives/C037R7Z21B5/p1674168289279759?thread_ts=1672858787.989389&cid=C037R7Z21B5), moved this logic into a hook, looking up emotion, I am not seeing any sort of function that can filter + join, am I looking at the wrong [package](https://www.npmjs.com/package/emotion)? Or did you mean lodash?